examples/topics/index.ipynb

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
I don't think there's any reason to commit the export/ directory, is there?  It's not used on the website; those images are just for me to supply to people who want screenshots.

Reply:
Ok that makes sense.  But why is it generated there at all? Shouldn't it be only in temporary files?

Reply:
Those aren't temporary files, they are exported versions -- they are meant to be left at the end, for people (typically me) to use outside of the notebook context.  We can disable them altogether, but for a time I was having people request screenshots all the time, so I had them generated as a side effect.